input/data/crop/c1f2_alfalfa_2020_rotation.json

```
@@ -0,0 +1,63 @@
+{
+    "crop_schedules": [
+        {
+            "crop_species": "alfalfa_hay",
+            "planting_years": [1997],
+            "pattern_repeat": 3,
+            "planting_skip": 6,
+            "harvesting_skip": 6,
```

```suggestion
            "harvesting_skip": 3,
```
I think this should only be three. Reason being that the final harvest is in 2001, and harvests should start again in 2005 (after it is planted the year before) - so skip 2002, 2003, 2004.